src/libs/actions/BankAccounts.js

```diff
@@ -179,6 +179,13 @@ function deletePaymentBankAccount(bankAccountID) {
                 value: {[bankAccountID]: {pendingAction: CONST.RED_BRICK_ROAD_PENDING_ACTION.DELETE}},
             },
         ],
+        successData: [
+            {
+                onyxMethod: CONST.ONYX.METHOD.MERGE,
+                key: `${ONYXKEYS.BANK_ACCOUNT_LIST}`,
+                value: {[bankAccountID]: null},
+            },
+        ],
```

Comment:
Can we add a short comment above the successData explaining why this is necessary? Setting the successData isn't common practice, so people may get confused.

```suggestion
        // comment here
        successData: [
            {
                onyxMethod: CONST.ONYX.METHOD.MERGE,
                key: `${ONYXKEYS.BANK_ACCOUNT_LIST}`,
                value: {[bankAccountID]: null},
            },
        ],
```